tests/test_rrt_dubins.py

```diff
@@ -3,8 +3,8 @@
 import os
 sys.path.append(os.path.dirname(os.path.abspath(__file__)) +
                 "/../PathPlanning/RRTDubins/")
-sys.path.append(os.path.dirname(os.path.abspath(__file__)) +
-                "/../PathPlanning/DubinsPath/")
+#sys.path.append(os.path.dirname(os.path.abspath(__file__)) +
```

Comment:
Why did you change it?

Reply:
Programm is working fine without this line.